ui-tests/src/test/java/org/openmrs/reference/FormTest.java

```diff
@@ -21,81 +21,83 @@
  * Created by nata on 24.06.15.
  */
 public class FormTest extends TestBase {
-	
-	private static String name = "newFormTest1";
-	private static String description = "description of new form";
-	private static String version = "1.2";
-	private HomePage homePage;
-	private AdministrationPage administrationPage;
-	private HeaderPage headerPage;
-	private ManageFormsPage manageForm;
-	private ManageHtmlFormsPage manageHtmlFormsPage;
-	private HtmlFormsPage htmlFormsPage;
-	private ClinicianFacingPatientDashboardPage patientDashboardPage;
-	
-	@Before
-	public void setUp() throws Exception {
-		homePage = new HomePage(page);
-		assertPage(homePage.waitForPage());
-		headerPage = new HeaderPage(driver);
-		administrationPage = new AdministrationPage(page);
-		manageForm = new ManageFormsPage(driver);
-		htmlFormsPage = new HtmlFormsPage(page);
-		manageHtmlFormsPage = new ManageHtmlFormsPage(page);
-		patientDashboardPage = new ClinicianFacingPatientDashboardPage(page);
-	}
-	
-	private void initiateFormPresence() throws InterruptedException {
-		manageHtmlFormsPage = homePage.goToAdministration().clickOnManageHtmlForms();
-		if (manageHtmlFormsPage.getElementsIfExisting(By.xpath("//*[contains(text(), '" + name + "')]")).isEmpty()) {
-			manageHtmlFormsPage.clickOnNewHtmlForm();
-			htmlFormsPage.createNewFormTest(name, description, version);
-		}
-		homePage.go();
-		homePage.goToManageForm();
-		if (!manageForm.addPresent()) {
+
```

Comment:
> is it in lign with openmrs code formatter as seen here https://wiki.openmrs.org/display/docs/Developer+How-To+Setup+And+Use+Eclipse

This is exactly what `OpenMRS` embraces  and at the same time recognized in software standards!

Reply:
> This is exactly what `OpenMRS` embraces and at the same time recognized in software standards!

FYI openmrs have its own code format different from other software standard.

Reply:
> FYI openmrs have its own code format different from other software standard.

I totally agree!

 AFAICT OpenMRS embraces quality software standards. Well every organization has its own convention though some conventions are shared across open source organizations since they are recognized internationally in the field of software development. 

Here are two samples of OpenMRS files within different modules. 
1. [WebModuleUtilTest](https://github.com/openmrs/openmrs-core/blob/master/web/src/test/java/org/openmrs/module/web/WebModuleUtilTest.java) from openmrs-core
2. [MergeVisitTest](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/master/ui-tests/src/test/java/org/openmrs/reference/MergeVisitsTest.java) from distro-referenceapplication
3. [APPUiActivatorTest](https://github.com/openmrs/openmrs-module-appui/blob/master/api/src/test/java/org/openmrs/module/appui/AppUiActivatorTest.java) from openmrs-module-appui
4. [ClinicianFacingPatientDashboardPage](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/master/ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java) from distro-referenceapplication.

Currently FormTest indention looks like [this](https://github.com/openmrs/openmrs-distro-referenceapplication/blob/master/ui-tests/src/test/java/org/openmrs/reference/FormTest.java) in master But after formatting as addressed in this PR the file is now formatted to this [indention.](https://github.com/kdaud/openmrs-distro-referenceapplication/blob/RATEST-188/ui-tests/src/test/java/org/openmrs/reference/FormTest.java)  

We glad to have @dkayiwa providing support and guidance at every point of need. I would love to hear from him on what he recommends about the code indention. 

Waiting to learn from his experience so that I became a consultant to myself before I make any PR to our repos.